core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -0,0 +1,220 @@
+/*
+ *  Copyright 2017 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+import java.util.function.IntPredicate;
+import java.util.function.Predicate;
+
+import javax.net.ssl.SSLEngine;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Ascii;
+
+import com.linecorp.armeria.client.SessionOption;
+import com.linecorp.armeria.client.retry.RetryingClient;
+
+import io.netty.channel.epoll.Epoll;
+import io.netty.handler.ssl.OpenSsl;
+
+/**
+ * The system properties that affect Armeria's runtime behavior.
+ */
+public final class Flags {
+
+    private static final Logger logger = LoggerFactory.getLogger(Flags.class);
+
+    private static final String PREFIX;
+
+    static {
+        String prefix = Flags.class.getPackage().getName();
```

Comment:
How about just inlining the String to make it clearer what the prefix actually is and more robust to moving this class around?